suites/src/main/java/module-info.java

```diff
@@ -0,0 +1,15 @@
+/** Runtime module of the suites. */
+module com.hedera.block.node.suites {
+    requires com.hedera.block.simulator;
+    requires com.swirlds.config.api;
+    requires com.swirlds.config.extensions;
+
+    // Require testing libraries
+    requires io.github.cdimascio;
+    requires org.junit.jupiter.api;
+    requires org.junit.platform.suite.api;
+    requires org.testcontainers;
+
+    exports com.hedera.block.suites to
+            org.junit.platform.commons;
+}
```

Comment:
Should we be checking in the `resources/block-0.0.3` binary files below?

Reply:
Not sure, if I understand fully the question, but currently as it is, it's working as expected, with simulator and tests, getting resources from correct folder.

Reply:
Should we be committing these binary files? They're part of the PR.
<img width="385" alt="image" src="https://github.com/user-attachments/assets/e7c283e7-921c-4956-a2f1-4ce42e58f6ae">

Reply:
Ah, yes. If not in this one, I would have to commit them in the next PR, where I add actual E2E Test between simulator and block-node, as a test sample.